components/inputs/input-time.js

```diff
@@ -131,6 +189,14 @@ class InputTime extends LitElement {
 		return undefined;
 	}
 
+	__onOpened() {
+		this.shadowRoot.getElementById('combo-container').setAttribute('aria-expanded', 'true');
```

Comment:
I would have expected dropdown to do this for you? I thought it looked for a parent with `d2l-dropdown-opener` and applied them automatically...